demos/demo.js

```diff
@@ -89,6 +89,7 @@ function Register(name, type, id) {
             case 'i8':  this.dataHex = utilIntToHex(value, 2); break;
             case 'i16': this.dataHex = utilIntToHex(value, 4); break;
             case 'i32': this.dataHex = utilIntToHex(value, 8); break;
+            case 'i64': this.dataHex = utilIntToHex(value, 16); break;
```

Comment:
This fails when `value >= (1 << 52)`. Additionally, note that `utilIntToHex` forces unsignedness via:
```js
    if (n < 0) {
        n += Math.pow(2, 32);
    }
```
This works only in integers up to 32 bits in size, but not beyond that. The signature of `utilIntToHex` needs to be changed and add support for `value_lo` and `value_hi`.